pgjdbc/src/test/java/org/postgresql/test/jdbc2/RefCursorTest.java

```diff
@@ -70,8 +84,17 @@ public void tearDown() throws SQLException {
   @Test
   public void testResult() throws SQLException {
     assumeCallableStatementsSupported();
+
+    testResult(Types.OTHER);
+    if (this.supportsRefCursor) {
+      testResult(Types.REF_CURSOR);
```

Comment:
Please, move that to another test method.
Test methods that have lots of assertions are harder to debug.

So a mere `@Test testResultRefCursor()` would make a difference